lib/edda-server/src/change_set_processor_task/materialized_view.rs

```diff
@@ -689,7 +689,16 @@ where
     MaterializedViewError: From<E>,
     MaterializedViewError: From<<T as TryInto<FrontendObject>>::Error>,
 {
-    let mv_kind = mv_kind.to_string();
+    let mv_kind:String = mv_kind.to_string();
+     let (from_checksum, _previous_data) = if let Some(previous_version) = frigg
+            .get_current_object(ctx.workspace_pk()?, ctx.change_set_id(), &mv_kind, &mv_id)
```

Comment:
Should be able to remove this from the new entity else case below, and re-use what is fetched here, since the only difference should be the comment.